source/IAccessibleHandler/__init__.py

```diff
@@ -61,12 +63,14 @@
 NAVRELATION_NODE_CHILD_OF = 0x1005
 NAVRELATION_EMBEDS = 0x1009
 
+if typing.TYPE_CHECKING:
+	import textInfos

```

Comment:
Any particular reason why this is not placed together with other imports at the top of file?

Reply:
No, I'll move it.